src/compiler/scala/tools/nsc/Parsing.scala

```diff
@@ -35,8 +36,8 @@ trait Parsing { self : Positions with Reporting =>
     }
 
     def incompleteHandled = incompleteHandler != null
-    def incompleteInputError(pos: Position, msg: String): Unit =
+    def incompleteInputError(pos: Position, msg: String, actions: List[CodeAction] = Nil): Unit =
```

Comment:
Re: https://github.com/scala/scala/pull/10406#issuecomment-1636053195 @SethTisue 
I guess we can turn this into overload instead of default param?

Reply:
I guess, but it seems unsatisfying that we would be so constrained by some random test code that Scalameta happens to have. I mean Scalameta isn't just any random compiler plugin, it's a core one. but still

Reply:
cc @tgodzik, wdyt?

Reply:
I think we release testkit only for the latest Scala version so that should be fine.

Reply:
ah okay 👍